ansible/setup-management-network.sh

```diff
@@ -41,13 +66,15 @@ if ! command -v ethtool; then
 fi
 echo
 
-# todo, add parameter to indicate whether remove br1 forcefully
-echo "STEP 5: Remove existed br1..."
-if ifconfig br1; then
+echo "STEP 5: Delete existed br1..."
+if [ "$DEL_EXISTED_BRIDGE" = true ] && ifconfig br1 >/dev/null 2>&1; then
```

Comment:
Sorry but not quite understand, do you mean "==" instead of "="?

I got this usage from https://stackoverflow.com/questions/2953646/how-can-i-declare-and-use-boolean-variables-in-a-shell-script and the suggestion from chatGPT

And with my locally test, it works well